test/Microsoft.Dnx.TestHost.Tests/ProjectTest.cs

```diff
@@ -21,12 +21,11 @@ public class ProjectTest
         public ProjectTest()
         {
             var libraryManager = PlatformServices.Default.LibraryManager;
-
             // Example: "C:\github\testing\test\Sample.Tests"
-            _testProjectPath = Path.GetDirectoryName(libraryManager.GetLibrary(projectName).Path);
+            _testProjectPath = "Not set"; // Path.GetDirectoryName(libraryManager.GetLibrary(projectName).Path);
         }
 
-        [Fact]
+        [Fact(Skip = "TestHost has to be redesigned to support the new project format")]
```

Comment:
Do we have an issue that can be linked here? Does that issue mean `TestHost` is useless at the moment? (We have a fair number of repos using `TestHost` don't we?)